src/features/bookings/components/BookingDetailsTicketContent.native.test.tsx

```diff
@@ -47,9 +47,27 @@ describe('BookingDetailsTicketContent', () => {
       getByTestId('qr-code')
     })
 
-    it('should not display the QR code when proDisableEventsQrcode is true', () => {
+    it('should display the QR code when proDisableEventsQrcode is true and subcategory is not in notQrCodeSubcategories', () => {
+      const { getByTestId } = render(
+        <BookingDetailsTicketContent booking={initialBooking} proDisableEventsQrcode={false} />
+      )
+      getByTestId('qr-code')
+    })
+
+    it('should not display the QR code when proDisableEventsQrcode is true and subcategory is in notQrCodeSubcategories', () => {
```

Comment:
Est-ce qu'on parlerait pas d'`event subcategory` plutôt que de `subcategory is in notQrCodeSubcategories` ?